docker/scripts/permissions-check.sh

```
@@ -0,0 +1,37 @@
+#!/bin/sh
+set -euo pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)